app/models/habit.rb

```diff
@@ -6,4 +6,23 @@ class Habit < ActiveRecord::Base
   validates :goal, presence: true
   validates :title, presence: true
   validates :active, inclusion: { in: [true, false] }
+
+  def self.most_missed
+    habits = Habit.all
+    highest_missed, second_missed, most_missed = 0, 0, {1 => '', 2 => ''}
```

Comment:
Do not use parallel assignment.<br>Space inside { missing.<br>Space inside } missing.